### PR TITLE
Tracking authorization request & network ID's

### PR DESCRIPTION
**Pull request by arnaumartifont**

### CODE REVIEW

File: Objective-C/ChartboostExample/ViewController.m [hunk 1]
```diff
@@ -27,6 +28,32 @@ - (void)viewDidLoad {
     [self log:self.logBeforeViewDidLoad];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    if (@available(iOS 14, *)) {
+        [ATTrackingManager requestTrackingAuthorizationWithCompletionHandler:^(ATTrackingManagerAuthorizationStatus status) {
```
[the remark this answers is not in the document]
Comment (daniel-barros) [reply Objective-C/ChartboostExample/ViewController.m:34]: Also, the logs now just say "Authorized" or "Denied". I think we could give a bit more context in the log message with something like:
`[self log:@"Tracking authorization status: Authorized"]`.
Further more, just to try to keep this file as short and simple as possible (it does many things already) maybe we could use a single line for all cases? Like
`[self log:@"Tracking authorization status: %li", status]`. (not sure that the %li thing is correct on the spot).

[the remark this answers is not in the document]
Comment (daniel-barros) [reply Objective-C/ChartboostExample/ViewController.m:34]: If we make some changes let's apply to the Swift project too

**Review by daniel-barros — CHANGES_REQUESTED** [review]
Overall looks good 👍 
Should we update the SDK version in the readme too?

File: Swift/ChartboostExample/ViewController.swift [hunk 2]
```diff
@@ -64,6 +70,25 @@ class ViewController: UIViewController, CHBInterstitialDelegate, CHBRewardedDele
 
     //MARK: - Class Methods
     
+    func requestPermission() {
+        if #available(iOS 14, *) {
+            ATTrackingManager.requestTrackingAuthorization { status in
+                switch status {
```
[the remark this answers is not in the document]
Comment (arnaumartifont) [reply Swift/ChartboostExample/ViewController.swift]: 👍

File: Objective-C/ChartboostExample/ViewController.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (arnaumartifont) [reply Objective-C/ChartboostExample/ViewController.m]: 👍

File: Objective-C/ChartboostExample/ChartboostExample-Info.plist [hunk 3]
```diff
@@ -57,5 +57,94 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSUserTrackingUsageDescription</key>
+	<string>App would like to access IDFA for tracking purposes</string>
```
[the remark this answers is not in the document]
Comment (arnaumartifont) [reply Objective-C/ChartboostExample/ChartboostExample-Info.plist]: 👍

File: Objective-C/ChartboostExample/ViewController.m [hunk 4]
```diff
@@ -27,6 +28,13 @@ - (void)viewDidLoad {
     [self log:self.logBeforeViewDidLoad];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+    //Showing an explanation on this permission to the user prior to making the request may lead to a better user experience and improved monetization for your app.
+    [self requestPermission];
```
Comment (daniel-barros) [inline Objective-C/ChartboostExample/ViewController.m:35]: Sorry just one more minor thing, we can rename this to something that indicates what this permission is?
Maybe `requestTrackingAuthorization`?

Comment (daniel-barros) [reply Objective-C/ChartboostExample/ViewController.m:35]: Same in Swift

Comment (arnaumartifont) [reply Objective-C/ChartboostExample/ViewController.m]: 👍

**Review by daniel-barros — APPROVED** [review]

**Comment by daniel-barros** [comment]
Very nice @arnaumartifont . Let's make sure we squash commits when merging, name the commit to something like "Implemented Chartboost SDK 8.3.0 support", and more important, don't merge yet! 😛

**Comment by daniel-barros** [comment]
Hey @arnaumartifont I think we've forgotten to update the podfile

**Review by daniel-barros — CHANGES_REQUESTED** [review]
Hey, besides the podfile issue there's something more critical. 
The dispatch main we do on the Swift view controller is not properly placed. we need to apply it when we call the log function, otherwise we crash the app

File: Swift/ChartboostExample.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -379,7 +379,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.5;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (daniel-barros) [inline Swift/ChartboostExample.xcodeproj/project.pbxproj:382]: This is a good catch. Can you set it to iOS 10.0 though? iOS 9 is being dropped with the latest SDK release

Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:382]: Yes!, was about to ask that, i'll update both apps to iOS 10 🙌

Comment (arnaumartifont) [reply Swift/ChartboostExample.xcodeproj/project.pbxproj:382]: also in the pods files

**Review by daniel-barros — APPROVED** [review]

**Pull request merged**